<launch>
  <include file="$(find jsk_2015_05_baxter_apc)/launch/baxter_right_hand_qrcode_detect.launch"></include>
  <node pkg="jsk_2015_05_baxter_apc" type="speak_qrcode_contents.py" name="speak_qrcode_contents"></node>
  <node pkg="jsk_2015_05_baxter_apc" type="qrcode-pos-server.l" name="qrcode_pos_server"></node>
  <node pkg="jsk_2015_05_baxter_apc" type="picking_up_demo.l" name="move_right_arm_service"></node>
  <!-- <node pkg="jsk_2015_05_baxter_apc" type="move_right_arm_server.l" name="move_right_arm_server"></node> -->
  <node launch-prefix="xterm -e" pkg="jsk_2015_05_baxter_apc" type="qrcode-pos-server.l" name="qrcode_pos_server"></node>
  <node pkg="jsk_2015_05_baxter_apc" type="picking_up_demo.l" name="move_right_arm_server"></node>
  <node pkg="jsk_2015_05_baxter_apc" type="get-item.l" name="get_item"></node>
  <node pkg="jsk_2015_05_baxter_apc" type="release-item.l" name="release_item"></node>
  <!-- <node pkg="baxter_interface" type="head_action_server.py" name="rsdk_head_action_server"></node> -->
</launch>
